db-migration/run.sh

```diff
@@ -11,7 +11,7 @@ fi
 
 echo "Setting up conduit into Gov PaaS DB and making dump file"
 
-if cf conduit ${CF_SERVICE} -- pg_dump -v -j 2 --file dumpfile --no-acl --encoding utf8 --clean --no-owner --if-exists -Fc; then
+if cf conduit ${CF_SERVICE} -- pg_dump -v --file dumpfile --no-acl --encoding utf8 --clean --no-owner --if-exists -Fc; then
```

Comment:
Can't use workers with the -Fc option (which is an option to use the custom file format for pg_dump)